soc/espressif/esp32c6/Kconfig.soc

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_ESP32C6
+	bool
+	select SOC_FAMILY_ESPRESSIF_ESP32
+
+config SOC_ESP32_C6_WROOM_1U_N4
+	bool
+	select SOC_ESP32C6
+
+config SOC_ESP32_C6_WROOM_1U_N8
+	bool
+	select SOC_ESP32C6
+
+config SOC_ESP32C6
+	bool
+	select SOC_SERIES_ESP32C6
+
+config SOC_SERIES
+	default "esp32c6" if SOC_SERIES_ESP32C6
+
+config SOC
+	default "esp32c6" if SOC_SERIES_ESP32C6
+
+config SOC_PART_NUMBER
+	default "ESP32_C6_WROOM_1U_N4" if SOC_ESP32_C6_WROOM_1U_N4
+	default "ESP32_C6_WROOM_1U_N8" if SOC_ESP32_C6_WROOM_1U_N8
+	default "ESP32C6" if SOC_ESP32C6
```

Comment:
I don't see how this can ever be selected, boards will select `SOC_ESP32_C6_WROOM_1U_N4` or `SOC_ESP32_C6_WROOM_1U_N8` right?